cmd/antrea-agent/options_windows.go

```diff
@@ -61,3 +66,119 @@ func (o *Options) checkUnsupportedFeatures() error {
 	}
 	return nil
 }
+
+func registerService() error {
+	p, err := getServicePath()
+	if err != nil {
+		return err
+	}
+	m, err := mgr.Connect()
+	if err != nil {
+		return err
+	}
+	defer m.Disconnect()
+
+	c := mgr.Config{
+		ServiceType:  windows.SERVICE_WIN32_OWN_PROCESS,
+		StartType:    mgr.StartAutomatic,
+		ErrorControl: mgr.ErrorNormal,
+		DisplayName:  "Containerd",
+		Description:  "Container runtime",
+	}
+
+	// Configure the service to launch with the arguments that were just passed.
+	args := []string{"--run-service"}
+	for _, a := range os.Args[1:] {
```

Comment:
Github Action is reminding you that `os` here is not defined. Besides `os/exec`, `os` should also be imported.

Reply:
Yeah importing it :+1: